controllers/ajax.py

```diff
@@ -77,6 +80,10 @@ def hsblog():
             logger.debug('correct {} incorrect {} skipped {} time {}'.format(request.vars.correct, request.vars.incorrect, request.vars.skipped, request.vars.time))
             logger.debug('Error: {}'.format(e.message))
 
+    # Produce a default result.
```

Comment:
Something here is breaking the unit tests.  From the tests folder can you run the run_tests script?  You can see the breakage on github where I have travis set up to automatically run the unit tests on all PR's

Reply:
Yes, I see the breakage in Travis. I'm porting the test shell script to a Python script, so I can run it on my machine. However, I'm at a conference this week (I presented two papers based around Runestone!), so it will probably be next week.